CHANGELOG.md

```diff
@@ -15,7 +24,7 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 ### Changed
 
 - The RPC endpoint `/v3/block_proposal` no longer will evaluate block proposals more than `block_proposal_max_age_secs` old
-- When a transaction is dropped due to replace-by-fee, the `/drop_mempool_tx` event observer payload now includes `new_txid`, which is the transaction that replaced this dropped transaction. When a transaction is dropped for other reasons, `new_txid` is `null`. [#5381](https://github.com/stacks-network/stacks-core/pull/5381)
+- When a transaction is dropped due to replauce-by-fee, the `/drop_mempool_tx` event observer payload now includes `new_txid`, which is the transaction that replaced this dropped transaction. When a transaction is dropped for other reasons, `new_txid` is `null`. [#5381](https://github.com/stacks-network/stacks-core/pull/5381)
```

Comment:
Hi are you trying to make me look bad by adding a typo to my changelog entry?

Reply:
fixed!